htdocs/lib/settings-dev.inc.php

```diff
@@ -105,7 +105,7 @@
 $sql_replacements['tmpdb'] = $tmpdbname;
 
 // safemode_zip-binary
-$safemode_zip = '/var/www/bin/phpzip.php';
+$safemode_zip = '/var/www/html/local/prodsys/phpzip.php';
```

Comment:
(This script so far has been running outside of the OC code tree because of safe mode restrictions. But as safe mode has been discarded, there is no longer a reason to do that.)